runtime/Cargo.toml

```diff
@@ -28,10 +28,6 @@ include_js_files_for_snapshotting = [
 name = "deno_runtime"
 path = "lib.rs"
 
-[[example]]
-name = "hello_runtime"
-path = "examples/hello_runtime.rs"
-
```

Comment:
```

[[example]]
name = "extension_with_esm"
path = "examples/extension_with_esm/main.rs"

[[example]]
name = "extension_with_ops"
path = "examples/extension_with_ops/main.rs"
```

Reply:
Added but what do you mean 'need to make these examples runnable'? If the examples follow proper directory layout they are autodiscovered buy cargo and are runnable. Or is your CI requires these properties to be specified explicitly?